src/composer.ts

```diff
@@ -198,7 +198,7 @@ class Composer<TContext extends Context> implements Middleware.Obj<TContext> {
     const handler = Composer.unwrap(middleware)
     return (ctx, next) => {
       // eslint-disable-next-line @typescript-eslint/no-misused-promises
-      setImmediate(handler, ctx, Composer.safePassThru())
+      setImmediate(handler, ctx, Composer.passThru())
       return next()
```

Comment:
```suggestion
return Promise.all([handler(ctx, anoop), next()])
```

Reply:
I see where you're going with this, but you cannot return an array here. Should the left or the right value be returned?
```js
return Promise.all([handler(ctx, anoop), next()]).then(([l, r]) => l)
```
OR
```js
return Promise.all([handler(ctx, anoop), next()]).then(([l, r]) => r)
```

Reply:
Oh wait, it's the right one of course

Reply:
Users are not supposed to rely on values returned by middleware.

Reply:
Exactly :shrug: